From 15c53981f78c1dedf8f9ec99c7c72fd6e99d1ea9 Mon Sep 17 00:00:00 2001
From: Claudiu Manoil <claudiu.manoil@nxp.com>
Date: Wed, 10 Apr 2019 19:24:28 +0300
Subject: [PATCH 694/706] mscc: felix: Fix whitespaces and minor coding style
 issues

Addresses checkpatch issues on latest commits.

Signed-off-by: Claudiu Manoil <claudiu.manoil@nxp.com>
(cherry picked from commit 9096291177520c4ce3115d990a602e3b0a529b92)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/net/ethernet/mscc/felix_board.c | 21 +++++++++++----------
 1 file changed, 11 insertions(+), 10 deletions(-)

diff --git a/drivers/net/ethernet/mscc/felix_board.c b/drivers/net/ethernet/mscc/felix_board.c
index adf34c931809..b4c45c20be7c 100644
--- a/drivers/net/ethernet/mscc/felix_board.c
+++ b/drivers/net/ethernet/mscc/felix_board.c
@@ -106,10 +106,10 @@ static struct resource felix_switch_res[] = {
 		.flags = IORESOURCE_MEM,
 	},
 	{
-		.start	= 0x0070000,
-		.end	= 0x00701ff,
-		.name	= "devcpu_gcb",
-		.flags	= IORESOURCE_MEM,
+		.start = 0x0070000,
+		.end = 0x00701ff,
+		.name = "devcpu_gcb",
+		.flags = IORESOURCE_MEM,
 	},
 	{
 		.start = 0x0100000,
@@ -224,7 +224,8 @@ static void felix_tx_hdr_set(struct sk_buff *skb, struct ocelot_port *port)
 
 	if (skb_shinfo(skb)->tx_flags & SKBTX_HW_TSTAMP &&
 	    port->tx_tstamp) {
-		u8 id = (port->ocelot->tstamp_id ++) % 4;
+		u8 id = (port->ocelot->tstamp_id++) % 4;
+
 		felix_set_ifh_rew_op(ifh, 0x3 | id << 3);
 	}
 }
@@ -482,6 +483,7 @@ static void felix_get_hwtimestamp(struct ocelot *ocelot, struct timespec64 *ts)
 {
 	/* Read current PTP time to get seconds */
 	u32 val = ocelot_read_rix(ocelot, PTP_PIN_CFG, TOD_ACC_PIN);
+
 	val &= ~(PTP_PIN_CFG_SYNC | PTP_PIN_CFG_ACTION_MASK | PTP_PIN_CFG_DOM);
 	val |= PTP_PIN_CFG_ACTION(PTP_PIN_ACTION_SAVE);
 	ocelot_write_rix(ocelot, val, PTP_PIN_CFG, TOD_ACC_PIN);
@@ -520,12 +522,11 @@ static void felix_tx_clean_work(struct work_struct *work)
 		if (!(val & SYS_PTP_STATUS_PTP_MESS_VLD)) {
 			/* Exit if tstamp had been handled or timeout */
 			if (ktime_after(ktime_get(), timeout) ||
-			    tstamp_handled) {
+			    tstamp_handled)
 				return;
-			} else {
-				udelay(1);
-				continue;
-			}
+
+			udelay(1);
+			continue;
 		}
 
 		tstamp_handled = true;
-- 
2.17.1

